src/components/TimeSlicing.vue

```diff
@@ -112,19 +112,49 @@ export default defineComponent({
       }
     }
 
-    function exportNetwork() {
+    function exportEdges() {
       if (network.value === null) {
         return;
       }
+      // Slice network in case 'Generate Slices' button
+      // not clicked or updated
+      sliceNetwork();
+      const { slicedNetwork } = store.state;
+
+      // Generate edge table data
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const edges: any[] = [];
+      slicedNetwork.forEach((slice) => {
+        const timeObj = { slice: slice.slice, timeStart: slice.time[0], timeFinish: slice.time[1] };
+        slice.network.edges.forEach((edge) => {
+          const rowObj = Object.assign(edge, timeObj);
+          edges.push(rowObj);
+        });
+      });
+
+      // Formate edge data for CSV
+      const separator = ',';
+      const keys = Object.keys(edges[0]);
+      const edgeTable = `${keys.join(separator)
+      }\n${
+        edges.map((edge) => keys.map((k) => {
+          let cell = edge[k] === null || edge[k] === undefined ? '' : edge[k];
+          cell = cell instanceof Date
+            ? cell.toLocaleString()
+            : cell.toString().replace(/"/g, '""');
```

Comment:
This replaces all `"` with `""`?

Reply:
Yup!

Reply:
Okay, I know that `"` is usually used to show strings with commas in them for csvs so this could break some stuff. You'll have to play around with it later and see if it breaks.

Reply:
Noted, thanks Jack!